views/randomBeer.hbs

```diff
@@ -0,0 +1,8 @@
+<div>
+    {{#each beersFromApi}}
+        <img src="{{this.image_url}}" alt="beers">
+        {{this.name}}
```

Comment:
You can't forget to put the information that you are receiving from the API, inside it's designed tags. Since `this.name`, `this.description` and `this.tagline` are strings, you need to insert the response inside some `<p> or <h1>` (paragraphs or heading tags).